draft-ietf-quic-transport.md

```diff
@@ -1162,27 +1162,29 @@ An endpoint MAY use the following transport parameters:
 
 initial_max_streams_bidi (0x0002):
 
-: The initial maximum streams parameter contains the initial maximum number of
-  application-owned bidirectional streams the peer may initiate, encoded as an
-  unsigned 16-bit integer.  If this parameter is absent or zero,
-  application-owned bidirectional streams cannot be created until a
+: The initial maximum bidirectional streams parameter contains the initial
+  maximum number of application-owned bidirectional streams the peer may
+  initiate, encoded as an unsigned 16-bit integer.  If this parameter is absent
+  or zero, application-owned bidirectional streams cannot be created until a
   MAX_STREAM_ID frame is sent.  Note that a value of 0 does not prevent the
-  cryptographic handshake stream (that is, stream 0) from being used. For
-  example, a value of 0x05 would be translated to a bidirectional stream ID of
-  20 by a client or 17 by a server.  Setting this parameter is equivalent to
-  sending a MAX_STREAM_ID ({{frame-max-stream-id}}) immediately after completing
-  the handshake containing the resulting Stream ID.
+  cryptographic handshake stream (that is, stream 0) from being used. Setting
+  this parameter is equivalent to sending a MAX_STREAM_ID
+  ({{frame-max-stream-id}}) immediately after completing the handshake
+  containing the resulting Stream ID. For example, a value of 0x05 would be
```

Comment:
"resulting" doesn't make sense, because you don't describe a process that might produce a result.  Try "corresponding".